lib/chef/knife/zero_converge.rb

```diff
@@ -20,6 +20,12 @@ class ZeroConverge < Chef::Knife::BootstrapSsh
       self.options = Ssh.options.merge(self.options)
       self.options[:use_sudo_password] = Bootstrap.options[:use_sudo_password]
 
+      # fatal by default Ref: https://github.com/chef/chef/pull/6781
+      if self.options[:duplicated_fqdns]
+        self.options[:duplicated_fqdns][:description] = 'Behavior if FQDNs are duplicated, fatal by default'
```

Comment:
__[RuboCop]__

Metrics/LineLength: Line is too long. [108/100]


<!-- Comment by [SideCI](https://sideci.com) -->